clang/lib/3C/ProgramInfo.cpp

```diff
@@ -774,9 +805,9 @@ CVarOption ProgramInfo::getVariable(clang::Decl *D, clang::ASTContext *C) {
     // Get corresponding FVConstraint vars.
     FVConstraint *FunFVar = getFuncFVConstraint(FD, C);
     assert(FunFVar != nullptr && "Unable to find function constraints.");
-    return CVarOption(*FunFVar->getParamVar(PIdx));
-  }
-  if (FunctionDecl *FD = dyn_cast<FunctionDecl>(D)) {
+    return CVarOption(*FunFVar->getInternalParam(PIdx));
+
+  } else if (FunctionDecl *FD = dyn_cast<FunctionDecl>(D)) {
```

Comment:
Is this `else` just cleanup, or is it ruling out an actual (and bogus) path? Just curious.

Reply:
The else was removed by the clang-tidy formatting, but I added it again while editing. I've removed it again.